Sources/Classes/PDFViewController.swift

```diff
@@ -84,8 +84,17 @@ public final class PDFViewController: UIViewController {
     /// UIBarButtonItem used to override the default action button
     fileprivate var actionButton: UIBarButtonItem?
     
+    /// Default background color applied both to the view and the collectionView.
+    fileprivate var defaultBackgroundColor: UIColor {
```

Comment:
I think they did this because since the class is UIColor, "white" is clearly a color. "PDFViewController.default = .white" doesn't have the same effect, but the "backgroundColor" var you suggested down there is a good option.